tests/test_xsd.py

```diff
@@ -71,17 +83,21 @@ def test_xsd():
     """)
 
     xml_doc = run_and_parse(testdir).xml_doc
-    xmlschema = etree.XMLSchema(etree.parse(get_xsd()))
+    xmlschema = etree.XMLSchema(etree.parse(xmlpf._get_xsd()))
```

Comment:
Since an external program is accessing this method it should be marked as public.